index.js

```diff
@@ -1,6 +1,30 @@
 
 import { NativeModules } from 'react-native';
+import { Buffer } from 'buffer';
 
 const RNUportSigner = NativeModules.RNUportSignerModule;
 const RNUportHDSigner = NativeModules.RNUportHDSignerModule;
-export { RNUportSigner, RNUportHDSigner };
+const DerivationPathRoot = RNUportHDSigner.UPORT_ROOT_DERIVATION_PATH;
```

Comment:
To avoid confusion, this const should specify that the root is uport specific.
The logical derivation root is `m/` but the uPort root is much deeper and only considered a root in the uPort app space.

If it's easier, use `RNUportHDSigner.UPORT_ROOT_DERIVATION_PATH` directly as the default value passed to the signer.